### PR TITLE
Adds option to ignore master calibration files

### PR DESCRIPTION
**Pull request by juanep97**
This PR:
 - adds a `flags` field to MasterBias, MasterDark and MasterFlat models.
 - adds 'IGNORE' as a possible flag to these flags fields.
 - adds the possibility of setting and unsetting this flag in bulk in the admin change list:
 
imagen

### CODE REVIEW

File: iop4admin/modeladmins/masterbias.py [hunk 1]
```diff
@@ -43,8 +44,8 @@ def night(self, obj):
...
     def get_built_from(self, obj):
         self.allow_tags = True
-        link_L = list()
-        for rawfit in obj.rawfits.all():
-            url = reverse('iop4admin:%s_%s_changelist' % (RawFit._meta.app_label, RawFit._meta.model_name)) + f"?id={rawfit.id}"
-            link_L.append(rf'<a href="{url}">{rawfit.id}</a>')
-        return mark_safe(", ".join(link_L))
\ No newline at end of file
+        
+        ids_str_L = [str(rf.id) for rf in obj.rawfits.all()]
```
Comment (morcuended) [inline iop4admin/modeladmins/masterbias.py:48]: what does L stand for here? a list? I suggest naming the variable more explicitly?

File: iop4admin/modeladmins/masterdark.py [hunk 2]
```diff
@@ -51,8 +53,8 @@ def get_masterbias(self, obj):
...
-        for rawfit in obj.rawfits.all():
-            url = reverse('iop4admin:%s_%s_changelist' % (RawFit._meta.app_label, RawFit._meta.model_name)) + f"?id={rawfit.id}"
-            link_L.append(rf'<a href="{url}">{rawfit.id}</a>')
-        return mark_safe(", ".join(link_L))
\ No newline at end of file
+        
+        ids_str_L = [str(rf.id) for rf in obj.rawfits.all()]
+        a_href = reverse('iop4admin:%s_%s_changelist' % (RawFit._meta.app_label, RawFit._meta.model_name)) + "?id__in=%s" % ",".join(ids_str_L)
+        a_text = ", ".join(ids_str_L)
+        return mark_safe(f'<a href="{a_href}">{a_text}</a>')
```
Comment (morcuended) [inline iop4admin/modeladmins/masterdark.py:60]: this part seems the same as in masterbias and masterflat. Can the duplication be avoided?

**Review by morcuended — APPROVED** [review]
looks good to me, just some very minor comments

File: iop4admin/modeladmins/masterdark.py (hunk 2 above)
Comment (juanep97) [reply iop4admin/modeladmins/masterdark.py]: True. Refactored.

**Comment by juanep97** [comment]
Rebased on origin/main.

**Review by morcuended — APPROVED** [review]

**Pull request merged**